<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="Lenovo" failures="1" tests="1" name="Test(failed)" time="0.131" errors="0" timestamp="2024-08-19T09:42:55 IST">
  <testcase classname="org.openms.demo.test.OpenMrsTestNGTest2" name="activateVisitsAndAddAttachment" time="0.033">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.openms.demo.test.OpenMrsTestNGTest2.activateVisitsAndAddAttachment(OpenMrsTestNGTest2.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- activateVisitsAndAddAttachment -->
  <testcase classname="org.openms.demo.test.OpenMrsTestNGTest2" name="findPatient" time="0.0">
    <ignored/>
  </testcase> <!-- findPatient -->
  <testcase classname="org.openms.demo.test.OpenMrsTestNGTest2" name="registerPatient" time="0.0">
    <ignored/>
  </testcase> <!-- registerPatient -->
  <testcase classname="org.openms.demo.test.OpenMrsTestNGTest2" name="deletePatient" time="0.0">
    <ignored/>
  </testcase> <!-- deletePatient -->
</testsuite> <!-- Test(failed) -->
